cypress/integration/multi-pdf/navigations.ts

```diff
@@ -7,13 +7,15 @@ describe('Multi PDF navigation', () => {
     cy.findByRole('button', { name: 'Settings' }).click();
     cy.findByRole('radio', { name: 'Paginated' }).should('be.checked');
     cy.findByRole('button', { name: 'Table of Contents' }).click();
-    cy.findByRole('menuitem', { name: 'Foreword' }).click();
+    cy.findByRole('listitem', { name: 'Foreword' }).click();
     cy.wait(5000);
     cy.findByText('Anthropology without Informants', { timeout: 10000 }).should(
       'be.visible'
     );
     cy.findByRole('button', { name: 'Next Page' }).click();
-    cy.findByText('Foreword', { timeout: 10000 }).should('exist');
+    cy.get('div[class="react-pdf__Page__textContent"]').within(() => {
```

Comment:
In RTL philosophy we don't normally use class selectors - is this ok because we don't expect `react-pdf` classnames to change, or because there's really no way to get this content with role selectors, or both?

Reply:
@Toxiapo this one for you : )

Reply:
Yeah there's only a single instance of `react-pdf__Page__textContent` so this should be fine, the reason that it's currently wrapped with `.within` was that the cypress was complaining about 'multiple matches', which make sense now that the TOC is always on the page.

I think the better test is to test a paragraph instead of just the title of the page. Will post an update now!